LayoutTests/css2.1/t1202-counter-09-b-expected.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN">
+<html>
+ <head>
+  <title>CSS 2.1 Test Suite: content: counter(c, georgian)</title>
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#propdef-content">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/syndata.html#counter">
+  <link rel="help" href="http://www.w3.org/TR/CSS21/generate.html#counter-styles">
+  <meta http-equiv="Content-Style-Type" content="text/css"></meta>
+  <style type="text/css">
```

Comment:
```suggestion
  <style>
```

Reply:
Done in [e2be840](https://github.com/WebKit/WebKit/commit/e2be84005a836155a72ee3465daaba6c17546dc9).